src/modules/users/interfaces/user.ts

```diff
@@ -0,0 +1,10 @@
+import { Role } from '../../../enums/role';
+
+export interface User {
+  readonly _id: string;
```

Comment:
let's change field name to id

Reply:
btw, this `readonly` doesn't do any affect on your code in runtime.
only thing it does is preventing of reassigning when you build the code